pkg/apis/flowcontrol/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// +groupName=flowcontrol.apiserver.k8s.io
+
+package flowcontrol // import "k8s.io/kubernetes/pkg/apis/flowcontrol"
```

Comment:
Either omit the doc.go file, or provide a package comment here.

Reply:
I thought we were headed in the direction of putting all the generator comments in a `doc.go` file.

Reply:
The purpose of a doc.go file is to give humans information about the package. e.g. see here: https://blog.golang.org/godoc-documenting-go-code

Reply:
This has been resolved in this case, but not the other two.